### ISSUE TITLE
Simplify language selection in package document

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#508]
Ignoring #503 for now, the javascript xpath resolver also reports the expression to get the language of the summary is invalid:

> `/package/metadata/meta[@property="schema:accessibilitySummary"]/(@lang|ancestor::*/@lang)[last()]`

I haven't dug into what it specifically doesn't like, but I can see right away that it will never match the language and it's more complicated than needed for epub processing.

The big issue here is that it is looking for a `lang` attribute but the package document only allows `xml:lang`.

But there are also only two locations where you can get the language of a metadata element: on the element itself or from the root package element. You don't need to inspect the ancestor chain because the only other element in the ancestor chain is `metadata` and it's not allowed to have an `xml:lang` declaration.

In other words, for getting the language, you only need to check:

> `/package/metadata/meta[@property="schema:accessibilitySummary"]/@xml:lang`

or:

> `/package/@xml:lang`

**Comment by mattgarrish** [comment]
> I haven't dug into what it specifically doesn't like,

For anyone interested, it looks like it's any expression using features above xpath 1.0. Most of the processing is easily implemented in javascript, but there are a couple of sticking points.

**Comment by mattgarrish** [comment]
Combining the above, I was able to use this to get the language for the summary:

> `(/opf:package/opf:metadata/opf:meta[@property="schema:accessibilitySummary"]/@xml:lang | /opf:package/@xml:lang)[last()]`

**Comment by mattgarrish** [comment]
The preceding xpath is buggy. It needs to be specific to the first summary, so:

> (/opf:package/opf:metadata/opf:meta[@property="schema:accessibilitySummary"][1]/@xml:lang | /opf:package/@xml:lang)[last()]

For the techniques, you could still use the original but just be more specific about the ancestor:

> /package/metadata/meta[@property="schema:accessibilitySummary"]/(@lang|/package/@lang)[last()]
